src/app/containers/AssignmentDutyEditForm.tsx

```diff
@@ -23,17 +25,19 @@ const formConfig: ConfigProps<any, AssignmentDutyFormProps> = {
 
 export interface AssignmentDutyEditFormProps extends AssignmentDutyFormProps {
     id: IdType;
+    assignmentId: IdType;
```

Comment:
so we actually have this id already within the duty object so down below ⬇️  you should be able to do the following 

```ts
const {assignmentTitle} = getAssignment(initialAssignmentDuty.assignmentId)(state);
```